IncubatingIntegrationTest/TestBase.cs

```diff
@@ -3,11 +3,12 @@ namespace IncubatingIntegrationTest;
 public class TestBase
 {
     protected readonly SimpleCacheClient client;
-    protected readonly string cacheName = SimpleCacheClientFixture.CacheName;
+    protected readonly string cacheName;
     protected const uint defaultTtlSeconds = SimpleCacheClientFixture.DefaultTtlSeconds;
 
     public TestBase(SimpleCacheClientFixture fixture)
     {
         this.client = fixture.Client;
+	this.cacheName = fixture.CacheName;
```

Comment:
Nit: some weird formatting happening here

Reply:
Gah. Thank you. Fixed in [02e561e](https://github.com/momentohq/client-sdk-csharp/pull/105/commits/02e561ebfd9e54cff79b3ecc665475cb41c3f7db)